app/gui/src/controller/upload.rs

```diff
@@ -77,6 +77,8 @@ pub struct FileToUpload<DataProvider> {
 #[derive(Clone, Debug)]
 pub struct FileUploadProcess<DataProvider> {
     bin_connection:  Rc<binary::Connection>,
+    // See FIXME in upload_chunk method.
```

Comment:
I don't see any FIXME in `upload_chunk` method. Is this still necessary?

Reply:
It's a change committed to a wrong branch.

#6689 also contains the change, once it will be merged, here it should disappear from diff.